presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1757,6 +1757,10 @@ Map of Iceberg types to the relevant PrestoDB types:
     - ``TIME``
   * - ``TIMESTAMP``
     - ``TIMESTAMP``
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP_WITH_TIMEZONE``
+  * - ``STRING``
+    - ``VARCHAR``
```

Comment:
I think we also need the mapping for the table below (prestodb -> iceberg)?

Reply:
Actually I'm not sure about this. The below table contains...

```
  * - ``TIMESTAMP``
    - ``TIMESTAMP WITHOUT ZONE``
  * - ``TIMESTAMP WITH TIMEZONE``
    - ``TIMESTAMP WITH ZONE``
```

I suppose with and without zone just represent `isAdjustedUTC` in this case? Either way it looks like this part is covered. Maybe we should standardize the way we discuss these in the documentation though